platforms/jvm/plugins-application/src/integTest/groovy/org/gradle/integtests/ApplicationIntegrationSpec.groovy

```diff
@@ -62,8 +62,8 @@ class Main {
         run 'installDist'
 
         def builder = new ScriptExecuter()
-        builder.workingDir file('build/install/application/bin')
-        builder.executable "application"
+        builder.setWorkingDir(file('build/install/application/bin'))
+        builder.setExecutable("application")
```

Comment:
💅 Use `=`?

Reply:
We can use it yes, even though IntellIj doesn't recognize it correctly. Done